pkgs/development/compilers/blueprint/default.nix

```diff
@@ -1,23 +1,30 @@
-{ fetchFromGitLab
+{ dbus
+, diffutils
+, fetchFromGitLab
+, gcc
+, gnat
```

Comment:
Why these?

Reply:
Leftover from trying to fix some bugs. Is there a tool that automatically removes these unused packages?

Reply:
`nixpkgs-hammering` can warn you. Maybe some other linters can remove them as well.